schedule/kernel/ibtest-master.yaml

```diff
@@ -2,6 +2,9 @@ name:           ibtest-master
 description:    >
     The master node for the infiniband testsuite (hpc-testing)
 vars:
+    AGAMA: 1
+    AGAMA_PRODUCT_ID: SLES
+    INST_AUTO: yam/agama/auto/sles_default.jsonnet
     DESKTOP: textmode
     GRUB_TIMEOUT: 300
```

Comment:
SLE 16 agama don't support this variable at all. In selected profile` yam/agama/auto/sles_default.jsonnet` , there is  `stopOnBootMenu: true` which equals timeout -1.

Reply:
If we want to configure timeout, we need to use `timeout = integer`  in agama profile.